### PR TITLE
Hardware requirements for installation in Core docs

### PR DESCRIPTION
**Pull request by chris529**
Closes https://github.com/LiskHQ/lisk-docs/issues/29

### CODE REVIEW

File: docs/antora/modules/ROOT/pages/setup/binary.adoc [hunk 1]
```diff
@@ -27,6 +26,24 @@ The binary installation offers an easy and automated approach to setting up Lisk
...
+
+Machines with a minimum of 2 GB RAM for the Testnet.
+
+*Storage*:
+
+Machines with a minimum of 40 GB HDD.
+
+These recommendations are derived from the log level settings, in the event that the user needs to increase storage to prevent limited memory access and potential memory-related problems with a node.
+Furthermore, as more transactions are processed and added to the blockchain, the size of the blockchain increases over time and this directly affects the HDD storage requirements for a blockchain node.
+Hence, adhering to the above listed requirements is highly recommended.
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/binary.adoc:45]: This should be added as a footnote.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/antora/modules/ROOT/pages/setup/binary.adoc [hunk 2]
```diff
@@ -27,6 +26,24 @@ The binary installation offers an easy and automated approach to setting up Lisk
 
 // include::partial$create-a-new-user.adoc[]
 
+=== Hardware requirements
```
[the remark this answers is not in the document]
Comment (chris529) [reply docs/antora/modules/ROOT/pages/setup/binary.adoc:29]: Good point, now done 👍

File: docs/antora/modules/ROOT/pages/setup/binary.adoc [hunk 3]
```diff
@@ -27,6 +26,22 @@ The binary installation offers an easy and automated approach to setting up Lisk
...
+
+Memory::
+* Machines with a minimum of 4 GB RAM for the Mainnet.
+
+* Machines with a minimum of 2 GB RAM for the Testnet.
+
+Storage::
+* Machines with a minimum of 40 GB HDD.
+
+NOTE: These recommendations are derived from the log level settings, in the event that the user needs to increase storage to prevent limited memory access and potential memory-related problems with a node.
```
Comment (Tschakki) [inline docs/antora/modules/ROOT/pages/setup/binary.adoc:41]: Please turn all notes into footnotes

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good already!

File: docs/antora/modules/ROOT/pages/setup/binary.adoc (hunk 3 above)
Comment (chris529) [reply docs/antora/modules/ROOT/pages/setup/binary.adoc]: I have now converted all the relevant notes in each setup page into footnotes...That is a good point, as I agree a footnote here is more appropriate. 👍

**Review by Tschakki — APPROVED** [review]
LGTM 👍

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**